lib/susedistribution.pm

```diff
@@ -856,6 +857,12 @@ known uncheckable consoles are already ignored.
 =cut
 sub console_selected {
     my ($self, $console, %args) = @_;
+    if ((exists $testapi::testapi_console_proxies{'root-ssh'}) && $console eq 'root-console') {
+        $console = 'root-ssh';
+        my $ret = query_isotovideo('backend_select_console', {testapi_console => $console});
+        die $ret->{error} if $ret->{error};
+        $autotest::selected_console = $console;
+    }
```

Comment:
If it works this is good but it looks risky relying on the internals on os-autoinst which also happened e.g. in run_ltp.pm . I won't promise that this won't break again :)

I think it's a valid workaround for the time being but I suggest to not call it a real "fix". Probably you want to extend os-autoinst to foresee referencing the same console object as the target for multiple virtual ones from test scope or something like that.

Reply:
@okurz  I was experimenting with os-autoinst as well, this code part worked in testapi.pm too, but it was using distri specific consoles, so i decided to update susedistri.pm

```
sub select_console {
    my ($testapi_console, @args) = @_;
    bmwqemu::log_call(testapi_console => $testapi_console, @args);
    if ((exists $testapi_console_proxies{'root-ssh'}) && ($testapi_console == 'root-console')) {
       $testapi_console = 'root-ssh'
    }  
```

Reply:
@okurz created for my self https://progress.opensuse.org/issues/63859
Thank you for your recommendation!